.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  settings: {
+    react: {
+      version: "detect",
+    },
+  },
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: ["eslint:recommended", "plugin:react/recommended", "prettier"],
```

Comment:
> Do you by chance have a link about the prettier extension? I was able to find documentation on the other 2 as per #58 but haven't been able to find more on how this prettier extension works for ESLint

Sure do! https://prettier.io/docs/en/integrating-with-linters.html

Reply:
> As per [here](https://github.com/codefordallas/codefordallas.github.io/pull/49/files#r1225452267) you said because we are using this file that the following were removed from the `package.json` file:
> 
> - `react-app`
> 
> - `react-app/jest`
> 
> 
> 
> I assume the react plugin takes care of the 1st, what about Jest? I remembered Jest being a separate install & configuration from React.
> 
> 
> 
> **NOTE:** this also has a comment in the related file

Ah good point. I was planning on setting up jest and adding it into the CI in another PR. Didnt realize we already had it setup. I can revert if we want or just add it all in the test PR

Reply:
> I was planning on setting up jest and adding it into the CI in another PR. Didnt realize we already had it setup. I can revert if we want or just add it all in the test PR

If you're just going to add it in the test PR, then I'm fine pushing this as is.

> > Do you by chance have a link about the prettier extension? I was able to find documentation on the other 2 as per #58 but haven't been able to find more on how this prettier extension works for ESLint
> 
> Sure do! https://prettier.io/docs/en/integrating-with-linters.html

As to the link, thank you. That's the one I saw - I'll try to remember to link ones I find in the future and be more clear about the ask. I was looking more for [this](https://github.com/prettier/eslint-plugin-prettier#installation) (which I overlooked before I inquired - my bad)